README.md

```diff
@@ -1,2 +1,41 @@
-# license-checker-github-action
-🕵️ Audit your NPM dependencies and reject any forbidden license.
+# npm License Checker javascript action
+
+This action prints check licenses of npm dependencies.
```

Comment:
It does not only print the licenses but also makes the workflow to pass/fail depending on the result.